setup.py

```diff
@@ -121,6 +122,121 @@
     'Topic :: Scientific/Engineering :: Visualization',
     ]
 
+
+class NoseTestCommand(TestCommand):
+    """Invoke unit tests using nose after an in-place build."""
+
+    description = "Invoke unit tests using nose after an in-place build."
+    user_options = [
+        ("pep8-only", None, "pep8 checks"),
+        ("omit-pep8", None, "Do not perform pep8 checks"),
```

Comment:
I wrote this so long ago I can hardly remember if I changed those names  (I presume I did) . Probably I had the impression that the flags from the script runner were ambiguous.